src/bitcoin-cli.cpp

```diff
@@ -228,6 +230,61 @@ class BaseRequestHandler
     virtual UniValue ProcessReply(const UniValue &batch_in) = 0;
 };
 
+/** Process addrinfo requests */
+class AddrinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    static constexpr uint8_t m_networks_size{5};
+    const std::array<std::string, m_networks_size> m_networks{{"ipv4", "ipv6", "torv2", "torv3", "i2p"}};
```

Comment:
Does the following work?
```cpp
static constexpr std::array m_networks{"ipv4", "ipv6", "torv2", "torv3", "i2p"};
```

Reply:
Thanks! I think I hadn't dropped the type specifier, only the size arg. Took your suggestions and updated the file in 06c43201a714b0426cc68b2fd5c681e5df10af99.

Reply:
Cool 👍